x-pack/plugins/security_solution/public/detections/components/take_action_dropdown/index.tsx

```diff
@@ -107,11 +108,10 @@ export const TakeActionDropdown = React.memo(
     const isEvent = actionsData.eventKind === 'event';
 
     const isAgentEndpoint = useMemo(() => ecsData?.agent?.type?.includes('endpoint'), [ecsData]);
-
     const isEndpointEvent = useMemo(() => isEvent && isAgentEndpoint, [isEvent, isAgentEndpoint]);
 
-    const agentId = useMemo(
-      () => getFieldValue({ category: 'agent', field: 'agent.id' }, detailsData),
+    const osQueryAgentId = useMemo(
```

Comment:
Thanks for the change, makes it easier to understand 👍 
NIT: we usually refer to it as `Osquery` (capital O) so would be better to rename it to `osqueryAgentId`